mixer/cmd/build.go

```diff
@@ -157,13 +184,25 @@ var buildOldFormatCmd = &cobra.Command{
 			fail(err)
 		}
 
-		fmt.Println("Backing up trimmed groups.ini for future build")
-		// Copy the new trimmed groups.ini to use for the +20 build which will
-		// not have manifest created for fully deleted bundles
-		if err = b.CopyTrimmedGroupsINI(); err != nil {
+		// TODO: Inject any extra files (certs, etc) here
+		// if err := AddBundleExtrasFiles(); err != nil {...}
+
+		ver, err := strconv.Atoi(b.MixVer)
+		if err != nil {
 			fail(err)
 		}
 
+		// Build the +20 update so we don't have to switch tooling in between
+		err = b.BuildUpdate(buildFlags.prefix, ver, buildFlags.format, buildFlags.noSigning, !buildFlags.noPublish, buildFlags.keepChroot)
```

Comment:
Doesn't this need to be `newFormat` (or... `b.Config.Swupd.Format`) rather than `buildFlags.format`?